engine/src/main/java/com/datatorrent/stram/StreamingAppMasterService.java

```diff
@@ -604,6 +604,8 @@ protected void serviceInit(Configuration conf) throws Exception
   }
 
   public static final String PLUGINS_CONF_KEY = "apex.plugin.stram.plugins";
```

Comment:
Additionally, is it necessary to have a separate entry for setup and engine plugins? Can't platform distinguish them based on interfaces the class implements.